stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -331,6 +333,56 @@ private void getStyleFromFloat(
     styles.store(name, String.valueOf(value), isDefaultValue(value));
   }
 
+  private void getStyleFromObject(
+      View view,
+      String name,
+      Object value,
+      @Nullable ViewDebug.ExportedProperty annotation,
+      StyleAccumulator styles) {
+    if (annotation != null && annotation.deepExport() && value != null) {
+      Field[] fields = value.getClass().getFields();
+      try {
+        for (Field field : fields) {
+          field.setAccessible(true);
+          String propertyName = field.getName();
+
+          switch (propertyName) {
+            case "bottomMargin":
+              propertyName = "margin-bottom";
+              break;
+            case "topMargin":
+              propertyName = "margin-top";
+              break;
+            case "leftMargin":
+              propertyName = "margin-left";
+              break;
+            case "rightMargin":
+              propertyName = "margin-right";
+              break;
+            default:
+              propertyName =
+                  (annotation.prefix() == null ? "" : annotation.prefix()) + propertyName;
+              break;
+          }
+
+          ViewDebug.ExportedProperty subAnnotation =
+              field.getAnnotation(ViewDebug.ExportedProperty.class);
+
+          getStyleFromValue(
+              view,
+              convertViewPropertyNameToCSSName(propertyName),
+              field.get(value),
+              subAnnotation,
+              styles);
+        }
+      } catch (IllegalAccessException e) {
```

Comment:
let's wrap only the call(s) that might throw this exception, instead of wrapping almost the entire body of this method with it